silver/models/payment_methods.py

```diff
@@ -60,6 +62,11 @@ def as_list(cls):
 
     objects = InheritanceManager()
 
+    @property
+    def one_way_fields(self):
```

Comment:
Those are not tickets :))
Maybe something like `final_fields` or `locked_fields` or `imutable_fields`